src/js/components/reviewData/ReviewDataNarrative.jsx

```diff
@@ -81,11 +104,28 @@ export default class ReviewDataNarrative extends React.Component {
     }
 
     render() {
+        const onKeyDownHandler = createOnKeyDownHandler(this.downloadCommentsFile.bind(this));
         return (
             <div className="narrative-wrapper col-md-8">
                 <div className="gray-bg">
                     <h4>Add comments to files</h4>
-                    <ReviewDataNarrativeDropdown changeFile={this.changeFile.bind(this)} />
+                    <div className="row">
+                        <div className="col-md-7">
+                            <ReviewDataNarrativeDropdown changeFile={this.changeFile.bind(this)} />
+                        </div>
+                        <div className="col-md-5 pull-right">
+                            <div
+                                role="button"
+                                tabIndex={0}
+                                className="usa-da-download pull-right"
+                                onKeyDown={onKeyDownHandler}
+                                onClick={this.downloadCommentsFile.bind(this)}>
```

Comment:
Related to the above comment, this can become `this.downloadCommentsFile` when the binding is moved to the constructor.